src/main/java/gregtech/api/recipes/MatchingMode.java

```diff
@@ -0,0 +1,13 @@
+package gregtech.api.recipes;
+
+/**
```

Comment:
Is this doc Ok @LAGIdiot ? first time writing a doc ^^'

Reply:
I am not documentarist myself so it looks OK to me as it is straight forward and to the point.